sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/FilterPushdownSuite.scala

```diff
@@ -34,6 +34,7 @@ class FilterPushdownSuite extends PlanTest {
       Batch("Subqueries", Once,
         EliminateSubqueryAliases) ::
       Batch("Filter Pushdown", FixedPoint(10),
+        PushDownPredicate,
         CombineFilters,
         PushDownPredicate,
         BooleanSimplification,
```

Comment:
Original rule order in this test suite doesn't match the order in `Optimizer`. So the added test will pass even without this change. But if reordering it as same as in `Optimizer`, the added test will fail.